jekyll/_cci2/configuration-reference.md

```diff
@@ -135,7 +135,7 @@ version: 2.1
 executors:
   my-executor:
     docker:
-      - image: circleci/ruby:2.5.1-node-browsers
+      - image: cimg/ruby:3.0.3-browsers + Browser Tools Orb
```

Comment:
Is this meant to be here?

>  + Browser Tools Orb

Reply:
No, this should not be here. I went ahead and removed it.